domain/consensus/datastructures/pruningstore/pruningstore.go

```diff
@@ -15,14 +15,13 @@ var pruningSerializedUTXOSetkey = dbkeys.MakeBucket().Key([]byte("pruning-utxo-s
 type pruningStore struct {
 	pruningPointStaging      *externalapi.DomainHash
 	serializedUTXOSetStaging []byte
+	pruningPointCache        *externalapi.DomainHash
+	serializedUTXOSetCache   []byte
```

Comment:
I don't think we want this cache

Reply:
We spoke about this and decided to keep it for now.